### PR TITLE
3 get/api/categories

### PR DESCRIPTION
**Pull request by GWorsnop**
Completed ticket 3- get/api/categories. 
Included error handling for 404 - bad path.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,15 @@
-# Northcoders House of Games API
+NC-Games Project
 
-## Background
+I will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+SETUP
```
Comment (HeatherEver) [inline README.md:5]: might want to mention about 'npm install'

File: app/models/categories-model.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const connection = require("../../db/connection");
+
+exports.selectCategories = () => {
+  return connection.query("SELECT * FROM categories").then((result) => {
+    //console.log(result.rows);
```
Comment (HeatherEver) [inline app/models/categories-model.js:5]: delete console logs when you're done with them please

**Review by HeatherEver — COMMENTED** [review]
Nice work - just a couple of changes to make

**Comment by GWorsnop** [comment]
Applied Heather's improvements and merged.

**Pull request merged**